instrumentation/elasticsearch/Appraisals

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+appraise 'elasticsearch-7.12' do
```

Comment:
@fbogsany Does this [matrix build](https://github.com/open-telemetry/opentelemetry-ruby/blob/1dfea8504716d2a97c3ae1df6be0f523e7ac77e4/.github/workflows/ci.yml#L14) run appraisals without any additional configuration?

Reply:
You can see here the different configs for the different rubies -> https://github.com/open-telemetry/opentelemetry-ruby/blob/c4949ac8f51822d068c7a8de2a6baddf6970fcc6/.github/workflows/ci.yml#L17-L25

The `--include-appraisal` flag corresponds to these commands -> https://github.com/open-telemetry/opentelemetry-ruby/blob/c4949ac8f51822d068c7a8de2a6baddf6970fcc6/.toys/ci.rb#L63-L64